global-setup.e2e.ts

```diff
@@ -6,12 +6,22 @@ import stopLocalRegistry from './tools/scripts/stop-local-registry';
 
 export async function setup() {
   await globalSetup();
-  await startLocalRegistry();
-  execSync('npm install -D @code-pushup/cli@e2e');
-  execSync('npm install -D @code-pushup/nx-plugin@e2e');
-  execSync('npm install -D @code-pushup/eslint-plugin@e2e');
-  execSync('npm install -D @code-pushup/coverage-plugin@e2e');
-  await setupTestFolder('tmp/e2e');
+  const registry = await startLocalRegistry();
+  try {
+    execSync(`npm install -D @code-pushup/cli@e2e --registry=${registry}`);
+    execSync(
+      `npm install -D @code-pushup/nx-plugin@e2e --registry=${registry}`,
+    );
+    execSync(
+      `npm install -D @code-pushup/eslint-plugin@e2e --registry=${registry}`,
+    );
+    execSync(
+      `npm install -D @code-pushup/coverage-plugin@e2e --registry=${registry}`,
+    );
+    await setupTestFolder('tmp/e2e');
+  } catch (e) {
+    stopLocalRegistry();
+  }
```

Comment:
this way, we avoid leaving dozens of Verdaccio and Node instances in our machine :)